docker-compose.yml

```diff
@@ -150,8 +154,14 @@ services:
       dockerfile: ./docker/cilantro-frontend/Dockerfile
     ports:
       - "8081:80"
+    # not actually used in dev, since the frontend is started separately
     environment:
       VUE_APP_BACKEND_URI: *backend-uri
+      VUE_APP_ATOM_API_URL: *atomAPIURL
+      VUE_APP_ATOM_USER: *atom-user
+      VUE_APP_ATOM_PASSWORD: *atom-password
```

Comment:
Ich habe mir das mit den Env-Variablen in Vue noch mal genauer angeschaut. Die Environment-Variablen hier werden beim Builden des Docker-Containers verwendet und dann durch Webpack statisch mit in das JS gepackt. Das ist bei Username und Passwort natürlich gefährlich und wir sollten uns eine Alternative überlegen.

Mir fällt allerdings gerade keine andere Option ein, als die beiden Werte irgendwo im Frontend abzufragen. Das hätte auch den Vorteil, dass wir personalisierte Accounts für Atom nutzen könnten.

Reply:
Das machen wir dann aber gesondert würde ich sagen. Oder soll ich das hier im PR mitmachen?